### PR TITLE
Gingko upgrade; typo fixes; `sudo` > `su`; `apt-mark hold`

### LINKED ISSUE #18: pip and setuptools need to be upgraded
As pointed by @SamuelMarks ([post](https://groups.google.com/forum/#!topic/edx-code/yUwFGXiL0Vc)) pip and setuptools were upgraded in Ginkgo: https://github.com/edx/edx-platform/commit/041fede3340ac569b531d0d8cec1308734616e53#diff-7d02e9409f61f90c29a5518f737e3f55

We need to update the install instructions to reflect this.

### PR DESCRIPTION
**Pull request by SamuelMarks**
> Gingko upgrade; typo fixes; `sudo` > `su`; `apt-mark hold` for elasticsearch; `git clone -b`; remove pip & setuptools downgrade

Closes: https://github.com/regisb/openedx-install/issues/18

### CODE REVIEW

**Comment by regisb** [comment]
Hi @SamuelMarks, thanks for your PR! Did you test the upgrade from Ficus to Ginkgo? It would be great if it just worked out of the box! (I suspect it does, since I had no issue upgrading the docker containers in https://github.com/regisb/openedx-docker)

But why don't you want to pin `pip` and `setuptools` to a specific version? In the commit that you pointed to, I see that they are pinned to 9.0.1 and 34.0.2, respectively: https://github.com/edx/edx-platform/commit/041fede3340ac569b531d0d8cec1308734616e53#diff-7d02e9409f61f90c29a5518f737e3f55

Also, it's "Open edX", not "OpenEdx" :-/ See for instance https://open.edx.org/about-open-edx

**Comment by SamuelMarks** [comment]
Yeah I tested and got no issues.
Version of pip is set specifically in requirements, as you referenced. So no need for us to pin them.

Whoops, will edit that now 🖌

File: README.md [hunk 1]
```diff
@@ -75,14 +75,14 @@ Install base packages:
...
+    sudo -sHu edxapp
 
 Note that the `edxapp` user does not have sudo rights, so all following commands that are run with `sudo` will have to be run with a different user; for instance, the user with which you created the `edxapp` user. This is on purpose, as `edxapp` will be used to run the web services. But if you want to grant sudo rights to `edxapp`, run : `sudo usermod -a -G sudo edxapp`.
 
 Create folder in which everything will be installed:
 
-    sudo mkdir /opt/openedx
-    sudo chown edxapp:edxapp /opt/openedx
+    sudo mkdir /opt/Open edX
+    sudo chown edxapp:edxapp /opt/Open edX
```
Comment (regisb) [inline README.md]: I'd rather we did not insert spaces in the install path. Spaces need to be escaped and they make install instructions more complex.

File: README.md [hunk 2]
```diff
@@ -146,38 +147,31 @@ Install packages required by LMS/CMS:
 
 Clone repositories:
 
-    cd /opt/openedx
-    git clone https://github.com/edx/edx-platform.git
-    cd edx-platform/
-    git checkout open-release/ficus.master 
+    cd /opt/Open edX
+    git clone -b open-release/gingko.master https://github.com/edx/edx-platform.git
```
Comment (regisb) [inline README.md]: s/gingko/ginkgo/

File: README.md [hunk 3]
```diff
@@ -229,9 +223,9 @@ Edit [`config/lms/production.py`](./config/lms/production.py) and add it to `edx
 
 Edit [`config/cms/production.py`](./config/cms/production.py) and add it to `edx-platform/cms/envs/`.
 
-In production, don't forget to set appropriate urls in `/opt/openedx/lms.env.json` and `/opt/openedx/cms.env.json`. Note that you will need to choose different domain names for the LMS and the CMS. In particular, pick an appropriate value for the `SITE_NAME`.
+In production, don't forget to set appropriate urls in `/opt/Open edX/lms.env.json` and `/opt/Open edX/cms.env.json`. Note that you will need to choose different domain names for the LMS and the CMS. In particular, pick an appropriate value for the `SITE_NAME`.
 
-Usually, the lms is configured on the bare domain name ("myopenedx.com") or  on the `www` subdomain ("www.myopenedx.com"), while cms typically uses the `studio` subdomain ("studio.myopenedx.com").
+Usually, the lms is configured on the bare domain name ("myOpen edX.com") or  on the `www` subdomain ("www.myOpen edX.com"), while cms typically uses the `studio` subdomain ("studio.myOpen edX.com").
```
Comment (regisb) [inline README.md]: I think you did a global search & replace that went wrong :) The domain names should definitely not contain white space.

**Review by regisb — CHANGES_REQUESTED** [review]

**Comment by SamuelMarks** [comment]
Resolved

**Comment by regisb** [comment]
Alright, let me test this before we go further.

File: README.md [hunk 4]
```diff
@@ -129,6 +129,7 @@ Install elasticsearch 0.90.13 from the official repositories:
     sudo sh -c 'echo "deb http://packages.elasticsearch.org/elasticsearch/0.90/debian stable main" > /etc/apt/sources.list.d/elasticsearch.list'
     sudo apt update
     sudo apt install elasticsearch=0.90.13 openjdk-8-jdk
+    sudo apt-mark hold elasticsearch
```
Comment (regisb) [inline README.md:132]: :+1:

**Comment by SamuelMarks** [comment]
@regisb Any news on this PR?

**Comment by regisb** [comment]
@SamuelMarks haven't had time to test it yet! I just returned from holidays and will catch up on PRs/issues soon.

**Comment by regisb** [comment]
Thanks a lot @SamuelMarks for your work!

**Pull request merged**